crates/red_knot/src/module.rs

```diff
@@ -225,22 +225,23 @@ struct ModuleSearchPathInner {
     kind: ModuleSearchPathKind,
 }
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, is_macro::Is)]
 pub enum ModuleSearchPathKind {
-    // Project dependency
-    FirstParty,
+    /// "Extra" paths provided by the user in a config file, env var or CLI flag.
+    /// E.g. mypy's `MYPYPATH` env var, or pyright's `stubPath` configuration setting
+    Extra,
 
-    // e.g. site packages
-    ThirdParty,
+    /// Project dependency
```

Comment:
I find the word "dependency" here confusing. I wouldn't think of first-party code as a "dependency" -- the dependencies probably live in site-packages.

Reply:
I just carried over the pre-existing comment here, but yeah, agree that this isn't the best terminology. I'll fix it up in a followup.